.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build Firmware
 on: [push, pull_request]
 
 jobs:
-  build-urban:
+  build-all:
```

Comment:
Continuous Integration modified to build only a single project

Reply:
Is there any reason why we aren't building for all vehicles? It looks like the default builds for proto, but if we make changes to another vehicle, we'd lose the static analysis from those builds.

Reply:
Great point! Is there a way to pass in a constant to be able to compile 3 different sources?

Reply:
You can with compiler flags. The DeviceOS makefile accepts an argument `EXTRA_CFLAGS` which you can use to pass in macros to the compiler.

For example, if you wanted to pass in `CURRENT_VEHICLE` as 3 and to define `DEBUG`, you can do:

```bash
make all PLATFORM=boron APPDIR=/app/ TARGET_DIR=/$(OUTPUT_DIR) EXTRA_CFLAGS='-DCURRENT_VEHICLE=3 -DDEBUG=0'
```

Here's a [good reference](https://github.com/particle-iot/device-os/blob/develop/docs/build.md#variables).

Reply:
Done! I've made it so you can pass a CFLAG in for the vehicle or define it in the settings file. Both work, and I prefer the latter. CI now compiles all 3 vehicles again